### ISSUE TITLE
Features and Experience of Porting ProjFileTools 

### ISSUE THREAD

**Issue by Varorbc** [mhutch/MonoDevelop.MSBuildEditor#163]
I used to use the [ProjFileTools ](https://github.com/dotnet/ProjFileTools) extension, but it is no longer maintained. Today, I saw your extension on the [developer blog](https://devblogs.microsoft.com/visualstudio/experimental-msbuild-editor/). First of all, thank you for your work. If possible, I hope you can port the features and experience of ProjFileTools

**Comment by mhutch** [comment]
Thanks! The extension does use some logic from ProjFileTools to implement NuGet package name and version completion. However, the underlying XML editor is completely different so it's more a question of reimplementing things rather then porting them.

Are there any particular missing features or experiences you would like me to prioritize?

**Comment by mhutch** [comment]
Yes, displaying evaluated values is a known limitation I intend to fix at some point. It's nontrivial as we do not currently load the project into the MSBuild engine at all,  but I understand the value. Ideally I would like it to display the evaluated value at that point in the file as well as the overall evaluated value.

*Go to Definition* (F12) works on imports and targets. *Shift-F12* (Find References) works for items, properties and metadata in addition to targets and values.

It's not clear to me what the "definition" is for items and properties? Would you perhaps expect this to search for writes to the item/properties in the evaluation scope (i.e. outside of targets)?
